index.js

```diff
@@ -61,10 +70,10 @@ class APIClient {
         return response;
     }
 
-    async publish(target = 'default', token = this.fetchToken()) {
+    async publish({ target = 'default', deployPercentage }, token = this.fetchToken()) {
         const { extensionId } = this;
 
-        const request = await fetch(publishURI(extensionId, target), {
+        const request = await fetch(publishURI({ id: extensionId, target, deployPercentage }), {
```

Comment:
Let's have it accept the full name

```suggestion
        const request = await fetch(publishURI({ extensionId, target, deployPercentage }), {
```

Reply:
it's fixed too